dev/bench/bench.sh

```diff
@@ -426,13 +426,13 @@ create_opam() {
 
     if [ ! -z "$coq_native" ]; then opam install coq-native; fi
 
-    for package in coq-core coq-stdlib coqide-server coq; do
+    for package in coq-core rocq-init coqide-server coq; do
```

Comment:
Presumably, you need to remove `coq` here, since it now depends on a remote package.

Reply:
Indeed, this wouldn't prevent installing coq-stdlib since the repos are added just above but benching it would probably not work nicely as results would appear under coq. The coq-stdlib package should rather be added to the `coq_opam_packages` list on top of the script.